openslides_backend/models/checker.py

```diff
@@ -831,3 +839,18 @@ def get_to_generic_case(
         raise CheckException(
             f"The collection {foreign_collection} is not supported as a reverse relation in {collection}/{field}"
         )
+
+
+class InternalChecker(Checker):
```

Comment:
remove this also

Reply:
The class is not used anymore and you could remove it now

Reply:
My bad, missed it somehow